### PR TITLE
Extract config vars

### PR DESCRIPTION
**Pull request by lazzarello**
addresses #9 

My hack to remove http and ws port numbers from `proxy.js` and include in configuration, which can be managed by Ansible in a production environment.

I used the `{tokens}` object, which is confusing but it worked!

### CODE REVIEW

**Comment by aguestuser** [comment]
cool! imma quickly unhackify this by making a `network` config object in the application code and defining the ropsten ports there.

question to @lazzarello : do we eventually want ports defined as env vars by ansible (and picked up by application code configs later)? or is hard-coding them in application code configs okay?

**Comment by aguestuser** [comment]
@lazzarello : i moved the configs! ready for merge if it looks okay to you!

**Comment by aguestuser** [comment]
hmm... tests are failing in CI but passing on my machine...

**Comment by lazzarello** [comment]
Hard coding is okay since the values can be managed by an external system with templates.

Checking why CI breaks and merging.

File: test/proxy.spec.js [hunk 1]
```diff
@@ -26,7 +26,12 @@ describe("secure proxy", () => {
 
   after(() => {
     proxy.close()
-    try { target.close() } catch {}
+    try { 
+      target.close(); 
+    }
+    catch {
```
Comment (kdunn926) [inline test/proxy.spec.js]: Shooting from the hip here but does it need to be `catch (e)` like [this](https://github.com/lazzarello/eth-router/pull/13/commits/2faf4b00012e8889a33055535007aea9c2f0a6c0#diff-0715aa1e2e64432ab65c30c1e431dbd8R7)?

**Pull request merged**